Process workflows for blog and social (#916)
2i2c-org/team-compass: communication/social.md

> We don't usually use social media to engage directly with other individuals or organizations, though it's not explicitly forbidden.

Reviewer: ```suggestion
We don't usually use 2i2c's social media accounts to engage directly with other individuals or organizations, though we encourage individual team members to interact as themselves. 
```

> We just have not had the resources to plan for this kind of engagement.

Reviewer: ```suggestion
We have not had the resources to plan extensive social engagement as an organization.
```
